Libraries/xcsdk/Sources/Configuration.cpp

```diff
@@ -26,14 +28,20 @@ Configuration(
 {
 }
 
-std::string Configuration::
-DefaultPath()
+std::vector<std::string> Configuration::
+DefaultPaths()
 {
+    std::vector<std::string> defaultPaths;
     if (char const *environmentPath = getenv("XCSDK_CONFIGURATION_PATH")) {
-        return std::string(environmentPath);
+        defaultPaths.push_back(std::string(environmentPath));
     } else {
-        return "/var/db/xcsdk_configuration.plist";
+        char const *homePath = getenv("HOME");
+        if (getuid() != 0 && homePath != nullptr) {
+            defaultPaths.push_back(std::string(homePath) + "/.xcsdk/xcsdk_configuration.plist");
```

Comment:
No particular strong reason.  My team does something similar with organizing custom Buck files -- we store data inside of ~/.buckd/...